test/models/developer_test.rb

```diff
@@ -79,6 +79,11 @@ class DeveloperTest < ActiveSupport::TestCase
     refute_includes developers, developers(:unavailable)
   end
 
+  test "visible scope filters developers with 'visible' search_status" do
+    developers = Developer.visible
+    assert_includes developers, developers(:invisible)
+  end
+
```

Comment:
This test feels backwards, shouldn't the assertion check if the invisible developer is _not_ included in the query?